src/commands/title-escrow/endorseChangeOfOwner.ts

```diff
@@ -0,0 +1,63 @@
+import { Argv } from "yargs";
+import { error, info, success, warn } from "signale";
+import { getLogger } from "../../logger";
+import { endorseChangeOfOwner } from "../../implementations/title-escrow/endorseChangeOfOwner";
+import { TitleEscrowEndorseChangeOfOwnerCommand } from "./title-escrow-command.type";
+import { withGasPriceOption, withNetworkAndWalletSignerOption } from "../shared";
+import { getEtherscanAddress } from "../../utils";
+
+const { trace } = getLogger("title-escrow:endorse-change-of-owner");
+
+export const command = "endorse-change-owner [options]";
+
+export const describe = "Endorses the change of owner of transferable record to another address";
+
+export const builder = (yargs: Argv): Argv =>
+  withGasPriceOption(
+    withNetworkAndWalletSignerOption(
+      yargs
+        .option("address", {
+          alias: "a",
+          description: "Address of the token registry that the transferable record was issued from",
+          type: "string",
+          demandOption: true,
+        })
+        .option("tokenId", {
+          description: "Hash of the transferable record",
+          type: "string",
+          demandOption: true,
+        })
+        .option("newOwner", {
+          description: "Address of the new owner of the transferable record",
+          type: "string",
+          demandOption: true,
+        })
+        .option("newHolder", {
+          description: "Address of the new holder of the transferable record",
+          type: "string",
+          demandOption: true,
+        })
+    )
+  );
+
+export const handler = async (args: TitleEscrowEndorseChangeOfOwnerCommand): Promise<void> => {
+  trace(`Args: ${JSON.stringify(args, null, 2)}`);
+  try {
+    info(
+      `Connecting to the registry ${args.address} and attempting to endorse the change of owner of the transferable record ${args.tokenId} to new owner at ${args.newOwner} and new holder at ${args.newHolder}`
+    );
+    warn(
+      `Please note that if you do not have the correct privileges to the transferable record, then this command will fail.`
+    );
+    const { transactionHash } = await endorseChangeOfOwner({
+      ...args,
```

Comment:
small stuff but can just pass in args if there isn't any modification on the values : endorseChangeOfOwner(args)

Reply:
👌